js/stores/windowStore.js

```diff
@@ -664,6 +664,8 @@ const doAction = (action) => {
           if (action.notify) {
             ipc.send('autofill-popup-hidden', action.tabId)
           }
+        } else {
+          windowState = windowState.delete('contextMenuDetail')
```

Comment:
I think we should just move `L663` here. Just like reverting the part of this commit https://github.com/brave/browser-laptop/commit/7fb2e4c30c98b592c00fc390648f99672a6caad7#diff-733d30fbf34816322d3f25dd59c1eb99L651